internal/flavors/benchmark/aws_org.go

```diff
@@ -114,17 +114,17 @@ func (a *AWSOrg) getAwsAccounts(ctx context.Context, log *logp.Logger, initialCf
 
 	accounts := make([]preset.AwsAccount, 0, len(accountIdentities))
 	for _, identity := range accountIdentities {
-		var memberCfg awssdk.Config
-		if identity.Account == rootIdentity.Account {
-			memberCfg = rootCfg
-		} else {
-			memberCfg = assumeRole(
-				stsClient,
-				rootCfg,
-				fmtIAMRole(identity.Account, memberRole),
-			)
-		}
-
+		// TODO(kuba): Add a comment explaining role assumption:
+		// - we're logged in into the main account, we use "cloudbeat-root" role
+		// - "cloudbeat-root" has limited permissions - account listing and role assumption
+		// - CF StackSets will install "cloudbeat-securityaudit" role where applicable
```

Comment:
Have you verified this part? Does it install the `cloudbeat-securityaudit` role in the management account if that's included in the organization unit the user has selected?